### ISSUE TITLE
Double-Bladed Scimitar is incorrectly included in the Items (SRD) compendium

### ISSUE THREAD

**Issue by aaclayton** [foundryvtt/dnd5e#966]
Originally in GitLab by @MrVauxs

The Double-Bladed Scimitar is in the SRD Compendium while it is not in the actual SRD Document.
Given UAWGE (Unearthed Arcana: Wayfinder's Guide to Eberron) is given as a source, it makes me worry that other items from Unearthed Arcana might have been added, which are not in the SRD license.

If I find any more items like this I will edit / add a comment to this Issue.

EDIT: Ctrl + F'ing the Items Compendium results in no other items with a source correlating to Unearthed Arcana.

Tl;dr Double-Bladed Scimitar is in the SRD Compendium where it shouldn't be. Most likely the only item that is wrongfully in the compendium.
